Few build errors: 

```
[2884/2959] Building CXX object Source/WebCore/CMakeFiles/WebCore.dir/platform/ultralight/PreserveSymbolsUltralight.cpp.o

FAILED: Source/WebCore/CMakeFiles/WebCore.dir/platform/ultralight/PreserveSymbolsUltralight.cpp.o 

/usr/bin/c++     @Source/WebCore/CMakeFiles/WebCore.dir/platform/ultralight/PreserveSymbolsUltralight.cpp.o.rsp -MD -MT Source/WebCore/CMakeFiles/WebCore.dir/platform/ultralight/PreserveSymbolsUltralight.cpp.o -MF Source/WebCore/CMakeFiles/WebCore.dir/platform/ultralight/PreserveSymbolsUltralight.cpp.o.d -o Source/WebCore/CMakeFiles/WebCore.dir/platform/ultralight/PreserveSymbolsUltralight.cpp.o -c /var/lib/jenkins/workspace/WebCore_PR-1/Source/WebCore/platform/ultralight/PreserveSymbolsUltralight.cpp

/var/lib/jenkins/workspace/WebCore_PR-1/Source/WebCore/platform/ultralight/PreserveSymbolsUltralight.cpp:27:3: error: no matching function for call to 'JSObjectMakeArrayBufferWithBytesNoCopy'

  JSObjectMakeArrayBufferWithBytesNoCopy(nullptr, nullptr, nullptr, nullptr, nullptr, nullptr);

  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

DerivedSources/ForwardingHeaders/JavaScriptCore/JSTypedArray.h:154:23: note: candidate function not viable: no known conversion from 'nullptr_t' to 'size_t' (aka 'unsigned long') for 3rd argument

JS_EXPORT JSObjectRef JSObjectMakeArrayBufferWithBytesNoCopy(JSContextRef ctx, void* bytes, size_t byteLength, JSTypedArrayBytesDeallocator bytesDeallocator, void* deallocatorContext, JSValueRef* exception) CF_AVAILABLE(10_12, 10_0);

                      ^

/var/lib/jenkins/workspace/WebCore_PR-1/Source/WebCore/platform/ultralight/PreserveSymbolsUltralight.cpp:28:3: error: no matching function for call to 'JSObjectMakeTypedArray'

  JSObjectMakeTypedArray(nullptr, nullptr, nullptr, nullptr);

  ^~~~~~~~~~~~~~~~~~~~~~

DerivedSources/ForwardingHeaders/JavaScriptCore/JSTypedArray.h:48:23: note: candidate function not viable: no known conversion from 'nullptr_t' to 'JSTypedArrayType' for 2nd argument

JS_EXPORT JSObjectRef JSObjectMakeTypedArray(JSContextRef ctx, JSTypedArrayType arrayType, size_t length, JSValueRef* exception) CF_AVAILABLE(10_12, 10_0);

                      ^

/var/lib/jenkins/workspace/WebCore_PR-1/Source/WebCore/platform/ultralight/PreserveSymbolsUltralight.cpp:29:3: error: no matching function for call to 'JSObjectMakeTypedArrayWithArrayBuffer'

  JSObjectMakeTypedArrayWithArrayBuffer(nullptr, nullptr, nullptr, nullptr);

  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

DerivedSources/ForwardingHeaders/JavaScriptCore/JSTypedArray.h:74:23: note: candidate function not viable: no known conversion from 'nullptr_t' to 'JSTypedArrayType' for 2nd argument

JS_EXPORT JSObjectRef JSObjectMakeTypedArrayWithArrayBuffer(JSContextRef ctx, JSTypedArrayType arrayType, JSObjectRef buffer, JSValueRef* exception) CF_AVAILABLE(10_12, 10_0);

                      ^

/var/lib/jenkins/workspace/WebCore_PR-1/Source/WebCore/platform/ultralight/PreserveSymbolsUltralight.cpp:31:8: error: redefinition of '_'

  auto _ = JSObjectMakeTypedArrayWithBytesNoCopy(nullptr, kJSTypedArrayTypeNone, nullptr, 0, nullptr, nullptr, nullptr);

       ^

/var/lib/jenkins/workspace/WebCore_PR-1/Source/WebCore/platform/ultralight/PreserveSymbolsUltralight.cpp:30:8: note: previous definition is here

  auto _ = JSObjectMakeTypedArrayWithArrayBufferAndOffset(nullptr, kJSTypedArrayTypeNone, nullptr, 0, 0, nullptr);

       ^

/var/lib/jenkins/workspace/WebCore_PR-1/Source/WebCore/platform/ultralight/PreserveSymbolsUltralight.cpp:32:8: error: redefinition of '_'

  auto _ = JSValueGetTypedArrayType(nullptr, nullptr, nullptr);

       ^

/var/lib/jenkins/workspace/WebCore_PR-1/Source/WebCore/platform/ultralight/PreserveSymbolsUltralight.cpp:30:8: note: previous definition is here

  auto _ = JSObjectMakeTypedArrayWithArrayBufferAndOffset(nullptr, kJSTypedArrayTypeNone, nullptr, 0, 0, nullptr);

       ^

5 errors generated.
```

It needs `#include "JavaScriptCore/JSTypedArray.h"` at top of file, and needs some non-duplicate names for the _ placeholders

--- a/Source/WebCore/platform/ultralight/PreserveSymbolsUltralight.cpp
+++ b/Source/WebCore/platform/ultralight/PreserveSymbolsUltralight.cpp
@@ -1,5 +1,6 @@
 #include "config.h"
 #include "JavaScriptCore/JavaScript.h"
+#include "JavaScriptCore/JSTypedArray.h"
 
 namespace WebCore {
 
@@ -24,12 +25,12 @@ WEBCORE_EXPORT void PreserveSymbols() {
   JSObjectGetTypedArrayByteOffset(nullptr, nullptr, nullptr);
   JSObjectGetTypedArrayBytesPtr(nullptr, nullptr, nullptr);
   JSObjectGetTypedArrayLength(nullptr, nullptr, nullptr);
-  JSObjectMakeArrayBufferWithBytesNoCopy(nullptr, nullptr, nullptr, nullptr, nullptr, nullptr);
-  JSObjectMakeTypedArray(nullptr, nullptr, nullptr, nullptr);
-  JSObjectMakeTypedArrayWithArrayBuffer(nullptr, nullptr, nullptr, nullptr);
-  auto _ = JSObjectMakeTypedArrayWithArrayBufferAndOffset(nullptr, kJSTypedArrayTypeNone, nullptr, 0, 0, nullptr);
-  auto _ = JSObjectMakeTypedArrayWithBytesNoCopy(nullptr, kJSTypedArrayTypeNone, nullptr, 0, nullptr, nullptr, nullptr);
-  auto _ = JSValueGetTypedArrayType(nullptr, nullptr, nullptr);
+  JSObjectMakeArrayBufferWithBytesNoCopy(nullptr, nullptr, 0, nullptr, nullptr, nullptr);
+  JSObjectMakeTypedArray(nullptr, kJSTypedArrayTypeNone, 0, nullptr);
+  JSObjectMakeTypedArrayWithArrayBuffer(nullptr, kJSTypedArrayTypeNone, nullptr, nullptr);
+  auto _0 = JSObjectMakeTypedArrayWithArrayBufferAndOffset(nullptr, kJSTypedArrayTypeNone, nullptr, 0, 0, nullptr);
+  auto _1 = JSObjectMakeTypedArrayWithBytesNoCopy(nullptr, kJSTypedArrayTypeNone, nullptr, 0, nullptr, nullptr, nullptr);
+  auto _2 = JSValueGetTypedArrayType(nullptr, nullptr, nullptr);
 }
 
 }

